mapie/tests/test_regression.py

```diff
@@ -361,6 +361,28 @@ def test_results_with_constant_sample_weights(strategy: str) -> None:
     np.testing.assert_allclose(y_pis1, y_pis2)
 
 
+@pytest.mark.parametrize("strategy", [*STRATEGIES])
+def test_results_with_constant_groups(strategy: str) -> None:
+    """
+    Test predictions when groups are None
+    or constant with different values.
+    """
```

Comment:
The addition of test results with non-constant groups has its place alongside this test to also cover this use case.

Reply:
Sure. I'll add some more tests.